src/clj/witan/send/model/prepare.clj

```diff
@@ -189,6 +176,63 @@
             (assoc coll [academic-year (states/join-need-setting need setting)] population))
           {} send-data))
 
+(defn get-filter-params [filter-what]
+  (into {} (filter #(every? (complement nil?) (vals (val %))) filter-what)))
+
+(defn build-pred [m]
+  (partial (:op (val m)) (:val (val m))))
+
+(def operators {:= =
+                :< <
+                :<= <=
+                :> >
+                :>= >=})
+
+(defn operator-key [filter-k]
```

Comment:
I don't think you need this function. I think you can use the operators map directly